content/docs/start/experiments/experiment-tracking.md

```diff
@@ -38,9 +38,13 @@ There are some examples below
 from dvclive.lightning import DVCLiveLogger
 
 ...
-
-trainer = Trainer(logger=DVCLiveLogger(save_dvc_exp=True))
-trainer.fit(model)
+with Live(save_dvc_exp=True) as live:
+    trainer = Trainer(
+        logger=DVCLiveLogger(save_dvc_exp=True),
```

Comment:
Do we need to pass `DVCLiveLogger(save_dvc_exp=True, experiment=live)` here (and similar for the other examples)?

Reply:
Yes, I didn´t notice.

Reply:
updated, please check!